docs/sql-ref-null-semantics.md

```diff
@@ -610,7 +610,8 @@ is a non-membership condition and returns TRUE when no rows or zero rows are
 returned from the subquery.
```

Comment:
It's not related to this PR, but could you fix `In otherwords` to `In other words` at line 607 while you are touching this section?

Reply:
Sure, done in f815452.